documentation/airport-guides/eddm.md

```diff
@@ -0,0 +1,105 @@
+### About Munich Airport (MUC | EDDM)
+Munich Airport (German: Flughafen München) is the international airport of Munich, the capital of Bavaria, Germany. It is the second-busiest airport in Germany in terms of passenger traffic after Frankfurt Airport, and the seventh-busiest airport in Europe. Munich Airport serves as a hub for Lufthansa including Lufthansa Regional and its Star Alliance partners. The airport is located 28.5 km (17.7 mi) northeast of Munich near the city of Freising.
+
+#### Statistics (2018)
+
+* **Total passengers**: *46,253,623*
+* **Aircraft movements**: *413,469*
+* **Cargo**: *368,377 tonnes*
+
+### About the facility
+There are four Area Control Centers in Germany: Bremen, Langen, Karlsruhe and München, operated by the Deutsche Flugsicherung (German Air Traffic Control). The terminal control service for EDDM is provided by the München ACC which is located at the airport. München ACC also provides terminal control services for Oberpfaffenhofen Airport (EDMO) and Augsburg Airport (EDMA) as well as air traffic services for en-route traffic within the München Flight Information Region (EDMM FIR).
+
+### Airspace
+The airspace encompasses a large area around Munich up to FL195. Big parts of the airspace are classified as Class C airspace (equivalent to American class B airspace). As long as one approach radar system (ASR) is in operation 3nm horizontal separation may be applied.
```

Comment:
As long as one approach radar system (ASR) is in operation **_,_** 3nm horizontal separation may be applied. (Add comma)